### PR TITLE
Lecture 7: Webdriver: login and add item to cart final

### PR DESCRIPTION
**Pull request by litvinyuk852**

### CODE REVIEW

File: Lecture7/src/test/java/ShopAddToCartTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    public void loginToSite() throws InterruptedException {
+        driver.get(siteUrl);
+        driver.manage().window().maximize();
+        Thread.sleep(2000);
+        driver.findElement(By.cssSelector(".ico-login")).click();
+        Thread.sleep(2000);
+        driver.findElement(By.name("data[User][email]")).sendKeys(userLogin);
+        driver.findElement(By.name("data[User][password]")).sendKeys(userPassword);
+        driver.findElement(By.cssSelector("button.g-button")).click();
+        Thread.sleep(2000);
```
Comment (acherkas) [inline Lecture7/src/test/java/ShopAddToCartTest.java]: where is assert of success of login?

File: Lecture7/src/test/java/ShopAddToCartTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    @Test(dependsOnMethods = "loginToSite")
+    public void addItemToCart() throws InterruptedException {
+        driver.findElement(By.cssSelector(".nav__item:nth-child(3) .g-pseudo_href")).click();
+        driver.findElement(By.cssSelector("#j-nav > dd:nth-child(4) > div > table:nth-child(1) > tbody > tr > td > dl > dd:nth-child(2) > a")).click();
+        driver.findElement(By.className("j-to_basket")).click();
+        driver.findElement(By.cssSelector(".cart__label > span")).click();
+        driver.findElement(By.className("j-basket-item")).isDisplayed();
```
Comment (acherkas) [inline Lecture7/src/test/java/ShopAddToCartTest.java]: why did you use it if you using in assertion the same?

File: Lecture7/src/test/java/ShopAddToCartTest.java (hunk 1 above)
Comment (litvinyuk852) [reply Lecture7/src/test/java/ShopAddToCartTest.java]: assert is added

File: Lecture7/src/test/java/ShopAddToCartTest.java (hunk 2 above)
Comment (litvinyuk852) [reply Lecture7/src/test/java/ShopAddToCartTest.java]: removed

**Pull request merged**